packages/browser/src/core/inspector/index.ts

```diff
@@ -0,0 +1,9 @@
+import { Inspector } from 'segment-inspector-core'
+
+// FIXME: Replace this with `globalThis` to support all environments
+const env = typeof window !== 'undefined' ? window : null
+
+export const inspectorHost: Inspector = {
```

Comment:
I added that wrapper that so that each caller wouldn't need to add optional chaining on every call eg: `inspectorHost?.trace(...)`, but on a second though, it's not a big deal. Will remove the wrapper.

```
if (typeof window !== 'undefined') {
   throw new Error("This can only be run in browser.")
}
```

We don't want to throw an errors that could be fatal for Analytics' operations. In case of Inspector, it is a "non-essential", so a silent fail is preferable.

Reply:
I see, so it was a wrapper for the sake of a noop